README.md

```diff
@@ -85,6 +86,19 @@ pip install dohq-artifactory==0.5.dev243
 ```
 
 # Usage
+## Enable logging 
```

Comment:
The section should be at the bottom. Logging is not the first step that developers do with the library, I'd say it's the last

Reply:
done